parlai/agents/rag/retrievers.py

```diff
@@ -259,7 +265,13 @@ def _init_tokenizer(
             ParlAI dictionary agent
         """
         if self.query_model in ['bert', 'bert_from_parlai_rag']:
-            return BertTokenizer.from_pretrained('bert-base-uncased')
+            try:
+                return BertTokenizer.from_pretrained('bert-base-uncased')
+            except ImportError or OSError:
```

Comment:
```suggestion
            except (ImportError, OSError):
```
I don't think or works here? I don't see anything in the python documentation about this syntax

Reply:
```python

In [1]: try:
   ...:     assert False
   ...: except OSError or AssertionError:
   ...:     print("Caught")
   ...:
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-1-c10757ab8e70> in <module>
      1 try:
----> 2     assert False
      3 except OSError or AssertionError:
      4     print("Caught")
      5

AssertionError:

In [2]: try:
   ...:     assert False
   ...: except (OSError, AssertionError):
   ...:     print("Caught")
   ...:
Caught

```